src/main/java/com/gooddata/lcm/LcmService.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (C) 2007-2018, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.lcm;
+
+import com.gooddata.AbstractService;
+import com.gooddata.GoodDataException;
+import com.gooddata.GoodDataSettings;
+import com.gooddata.account.Account;
+import com.gooddata.collections.MultiPageList;
+import com.gooddata.collections.Page;
+import com.gooddata.collections.PageRequest;
+import com.gooddata.collections.PageableList;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.util.UriComponentsBuilder;
+
+import java.net.URI;
+
+import static com.gooddata.util.Validate.notNull;
+
+/**
+ * Service, which provides access to lifecycle management objects.
+ */
+public class LcmService extends AbstractService {
+
+    /**
+     * Constructs service for GoodData Life Cycle Management.
+     *
+     * @param restTemplate RESTful HTTP Spring template
+     * @param settings settings
+     */
+    public LcmService(final RestTemplate restTemplate, final GoodDataSettings settings) {
+        super(restTemplate, settings);
+    }
+
+    /**
+     * Lists all {@link  LcmEntities} for given {@link Account}.
+     * Returns empty list in case there is no {@link LcmEntity}.
+     * Returns only first page if there's more instances than page limit. Use {@link PageableList#stream()} to iterate
+     * over all pages, or {@link MultiPageList#collectAll()} to load the entire list.
```

Comment:
@jimirocks @jakub-plichta why are we guiding user to use MultiPageList, but returning only PageableList? Is the MultiPageList meant to be used directly by end user?